CHANGES.md

```diff
@@ -1,112 +1,113 @@
 ## 0.0.37
```

Comment:
So it turns out the latest version of prettier does markdown, which is cool, but also means that every single line of this has been cleaned up.